test/testCographAlg.cpp

```diff
@@ -0,0 +1,33 @@
+#include <gtest/gtest.h>
+#include <list>
+
+#include "cograph_rec/CographAlg.hpp"
+#include "helpers.hpp"
+
+struct GraphRecognitionParameters {
+    int N;
+    std::list<std::pair<int, int>> E;
+    bool is_cograph;
+};
+
+class cograph_alg_test
```

Comment:
Convention: please use CamelCase as classnames and static function names, lowerCamelCase as public function names, and underscored_names as private function names